packages/basic-crawler/src/internals/basic-crawler.ts

```diff
@@ -663,7 +663,7 @@ export class BasicCrawler<Context extends CrawlingContext = BasicCrawlingContext
         await periodicLogger.log();
 
         const sigintHandler = async () => {
-            this.log.warning('Pausing... Press CTRL+C again to force exit. To resume, do: CRAWLEE_PURGE_ON_START=0 yarnstart');
+            this.log.warning('Pausing... Press CTRL+C again to force exit. To resume, do: CRAWLEE_PURGE_ON_START=0 yarn start');
```

Comment:
this change wasn't really on purpose, we should either keep `npm start` or do it conditionally based on existing lockfile type

Reply:
I don't have context on the reason this change was introduced, but fixing a typo in the meantime is sensible 👍